src/shared/sol-util-file.c

```diff
@@ -26,6 +26,7 @@
 #include <fcntl.h>
 #include <libgen.h>
 #include <limits.h>
+#include <sched.h>
```

Comment:
On 06-04-2016 11:28, Iván Briano wrote:
> In src/shared/sol-util-file.c
> <https://github.com/solettaproject/soletta/pull/1715#discussion_r58714521>:
>
>> @@ -26,6 +26,7 @@
>>  #include <fcntl.h>
>>  #include <libgen.h>
>>  #include <limits.h>
>> +#include <sched.h>
>
> On Wed, 06 Apr 2016 07:23:22 -0700, Gustavo Lima Chaves wrote: > @@
> -26,6 +26,7 @@ > #include <fcntl.h> > #include <libgen.h> > #include
> <limits.h> > +#include <sched.h> ifdef under linux as well?
> Or remove the whole thing? Why do we need to yield()? Most likely that
> function will be used from the main thread anyway.

Can it be... but i'd say that most likely isn't always. No strong 
opinion, though...

> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/1715/files/0d20a392a88ad51443271fd254faf37a46739bd0#r58714521>
>